src/value-converters/abstract-locale-value-converter.ts

```diff
@@ -3,12 +3,20 @@ import {EventAggregator} from "aurelia-event-aggregator";
 
 @autoinject()
 export abstract class AbstractLocaleValueConverter {
-    private static _locale:string = AbstractLocaleValueConverter.getDefaultLocale();
+    private static _locale:string;
```

Comment:
Should be non-static now? ... If not, what was the point?

Reply:
No, the goal was just to remove static methods, because they are evil and there is a way better way to call these.

Before:
```typescript
CurrencyValueFormatter.setLocale("de");
```

Now:
```typescript
const currencyValueFormatter = aurelia.container.get(CurrencyValueFormatter);
currencyValueFormatter.setLocale("de")
```

Reply:
But... the _behaviour_ has not changed. And before it was clearly communicated: _Of course_ a static function will serve/alter static state - what else!

Changing the function but not the field is an obfuscation: The "Now" does not look like something that would affect a second instance - but will do so.

I'll say that this is rather _more_ evil than less.

Reply:
I removed everything thats static.